src/sidebar/components/annotation.js

```diff
@@ -51,7 +52,7 @@ function Annotation({
   const toggleAction = threadIsCollapsed ? 'Show replies' : 'Hide replies';
   const toggleText = `${toggleAction} (${replyCount})`;
 
-  const shouldShowActions = !isSaving && !isEditing && !isNew(annotation);
+  const shouldShowActions = !isSaving && !isEditing && !isCollapsedReply;
```

Comment:
The `isNew` check should no longer be required now that `annotation` objects are more "normalized" via a single "initialization" step in the store—translation: by the time they are rendered in this component, `annotation` objects should be fully-qualified enough that this check no longer means anything.